src/Components/WebAssembly/WebAssembly/src/Microsoft.AspNetCore.Components.WebAssembly.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <Description>Build client-side single-page applications (SPAs) with Blazor running under WebAssembly.</Description>
-    <IsPackable>false</IsPackable>
```

Comment:
Oh wow. This would have been the opposite of fun on release day. Is p4 affected by this?

Reply:
Nope -- this bad merge was introduced yesterday afternoon.